src/Umbraco.Web.UI.Client/src/views/common/overlays/insert/insert.controller.js

```diff
@@ -0,0 +1,136 @@
+(function () {
+    "use strict";
+
+    function InsertOverlayController($scope) {
+
+        var vm = this;
+
+        if(!$scope.model.title) {
+            $scope.model.title = "Insert";
+        }
+
+        if(!$scope.model.subtitle) {
+            $scope.model.subtitle = "Choose what to insert into your template";
```

Comment:
Can we get this from a language file as opposed to hardcode in the controller @madsrasmussen please